rdmo/conditions/viewsets.py

```diff
@@ -31,6 +31,9 @@ class ConditionViewSet(CopyModelMixin, ModelViewSet):
         'target_option'
     )
 
+    def get_serializer_class(self):
```

Comment:
this `get_serializer_class` for the ViewSets could also go in a Mixin or not?

Reply:
Not really, since it is only one line in the method and it is specific to the particular viewset. It is just DRF's more flexible `serializer_class` attribute, much like `queryset = ...` and `def get_queryset(self):`.